### ISSUE TITLE
locale_dir should default to 'locales'

### ISSUE THREAD

**Issue by cocoatomo** [sphinx-doc/sphinx-intl#14]
The default value of `locale_dirs` is `['locales']` since Sphinx 1.5.
Therefore, for consistency and convenience, the name of default output directory of sphinx-intl should be a 'locales' directory.

**Comment by shimizukawa** [comment]
I see. It's definitely a bug..

**Comment by shimizukawa** [comment]
fixed by #15

**Comment by cocoatomo** [comment]
I tried following commands. Their result shows the issue has been resolved! thx!

```
$ pip install -U git+https://github.com/sphinx-doc/sphinx-intl.git
...
$ sphinx-intl update -p _build/gettext -l de
Create: locales/de/LC_MESSAGES/advanced.po
Create: locales/de/LC_MESSAGES/basics.po
Create: locales/de/LC_MESSAGES/index.po
```
